google-api-client-gson/pom.xml

```diff
@@ -26,15 +26,37 @@
         <artifactId>maven-jar-plugin</artifactId>
         <configuration>
           <archive>
+            <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
             <manifest>
               <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
             </manifest>
             <manifestEntries>
-              <Automatic-Module-Name>google.api.client.gson</Automatic-Module-Name>
+              <Automatic-Module-Name>google.api.client.jackson2</Automatic-Module-Name>
```

Comment:
This doesn't seem right. It should probably stay `gson`

Reply:
yep, done